setup.rst

```diff
@@ -37,32 +37,36 @@ In other words, your new app is ready!
 Running your Symfony Application
 --------------------------------
 
-On production, you should use a web server like Nginx or Apache
-(see :doc:`configuring a web server to run Symfony </setup/web_server_configuration>`).
-But for development, it's convenient to use the :doc:`Symfony Local Web Server <setup/symfony_server>`.
+On production, you should use a web server like Nginx or Apache (see
+:doc:`configuring a web server to run Symfony </setup/web_server_configuration>`).
+But for development, it's better to use the :doc:`Symfony Local Web Server <setup/symfony_server>`.
```

Comment:
Perhaps "recommended" would work here, since the alternative (`bin/console server:start`, which is what I'm familiar with) is planned to be deprecated in 4.4 and removed in 5.0.